packages/vue/test/components.js

```diff
@@ -0,0 +1,57 @@
+const components = {
```

Comment:
This is the a test utility file being consumed by the `mdx-provider.test.js` file